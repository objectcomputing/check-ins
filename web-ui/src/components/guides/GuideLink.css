.guide-info {
    text-align: left;
    margin: 0.75rem 0.25rem;
    display: flex;
    flex-direction: row;
    align-items: center;
}

.guide-link {
    margin-left: 1rem;
}

.guide-icon {
    vertical-align: middle;
}