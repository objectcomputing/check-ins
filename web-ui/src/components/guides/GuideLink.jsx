import PdfIcon from "@material-ui/icons/PictureAsPdf";
import React from "react";
import "./GuideLink.css";

const GuideLink = (props) => {
  const path = "/pdfs/";
  const fileName = props.name;
  let fullPath = path + fileName.split(" ").join("_") + ".pdf";
  return (
    <div className="guide-info">
      <div className="guide-icon">
        <PdfIcon fontSize="large" />
      </div>
      <div className="guide-link">
        <a href={fullPath} target="_blank" rel="noopener noreferrer">
          {fileName}
        </a>
      </div>
    </div>
  );
};

export default GuideLink;
