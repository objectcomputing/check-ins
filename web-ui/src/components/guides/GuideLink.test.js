import React from "react";
import GuideLink from "./GuideLink";


it("renders correctly", () => {
  snapshot(<GuideLink name="myFileName" />);
});
