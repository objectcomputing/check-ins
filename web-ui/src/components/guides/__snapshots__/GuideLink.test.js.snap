// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.emotion-0 {
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -ms-flex-pack: center;
  -webkit-justify-content: center;
  justify-content: center;
  position: relative;
  box-sizing: border-box;
  -webkit-tap-highlight-color: transparent;
  background-color: transparent;
  outline: 0;
  border: 0;
  margin: 0;
  border-radius: 0;
  padding: 0;
  cursor: pointer;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
  vertical-align: middle;
  -moz-appearance: none;
  -webkit-appearance: none;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: inherit;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: start;
  -ms-flex-pack: start;
  -webkit-justify-content: flex-start;
  justify-content: flex-start;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  position: relative;
  -webkit-text-decoration: none;
  text-decoration: none;
  width: 100%;
  box-sizing: border-box;
  text-align: left;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 16px;
  padding-right: 16px;
  -webkit-transition: background-color 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
  transition: background-color 150ms cubic-bezier(0.4, 0, 0.2, 1) 0ms;
}

.emotion-0::-moz-focus-inner {
  border-style: none;
}

.emotion-0.Mui-disabled {
  pointer-events: none;
  cursor: default;
}

@media print {
  .emotion-0 {
    -webkit-print-color-adjust: exact;
    color-adjust: exact;
  }
}

.emotion-0.Mui-focusVisible {
  background-color: rgba(0, 0, 0, 0.12);
}

.emotion-0.Mui-selected {
  background-color: rgba(25, 118, 210, 0.08);
}

.emotion-0.Mui-selected.Mui-focusVisible {
  background-color: rgba(25, 118, 210, 0.2);
}

.emotion-0.Mui-disabled {
  opacity: 0.38;
}

.emotion-0:hover {
  -webkit-text-decoration: none;
  text-decoration: none;
  background-color: rgba(0, 0, 0, 0.04);
}

@media (hover: none) {
  .emotion-0:hover {
    background-color: transparent;
  }
}

.emotion-0.Mui-selected:hover {
  background-color: rgba(25, 118, 210, 0.12);
}

@media (hover: none) {
  .emotion-0.Mui-selected:hover {
    background-color: rgba(25, 118, 210, 0.08);
  }
}

.emotion-1 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: start;
  -ms-flex-pack: start;
  -webkit-justify-content: flex-start;
  justify-content: flex-start;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  position: relative;
  -webkit-text-decoration: none;
  text-decoration: none;
  width: 100%;
  box-sizing: border-box;
  text-align: left;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 16px;
  padding-right: 16px;
}

.emotion-1.Mui-focusVisible {
  background-color: rgba(0, 0, 0, 0.12);
}

.emotion-1.Mui-selected {
  background-color: rgba(25, 118, 210, 0.08);
}

.emotion-1.Mui-selected.Mui-focusVisible {
  background-color: rgba(25, 118, 210, 0.2);
}

.emotion-1.Mui-disabled {
  opacity: 0.38;
}

.emotion-2 {
  -webkit-flex: 1 1 auto;
  -ms-flex: 1 1 auto;
  flex: 1 1 auto;
  min-width: 0;
  margin-top: 4px;
  margin-bottom: 4px;
}

.emotion-3 {
  margin: 0;
  font-family: "Roboto","Helvetica","Arial",sans-serif;
  font-weight: 400;
  font-size: 1rem;
  line-height: 1.5;
  letter-spacing: 0.00938em;
  display: block;
}

<a
  className="MuiButtonBase-root MuiListItem-root MuiListItem-gutters MuiListItem-padding MuiListItem-button emotion-0"
  href="/pdfs/myFileName.pdf"
  onBlur={[Function]}
  onClick={[Function]}
  onContextMenu={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  tabIndex={0}
  target="_blank"
>
  <li
    className="MuiListItem-root MuiListItem-gutters MuiListItem-padding emotion-1"
    disabled={false}
  >
    <div
      className="MuiListItemText-root emotion-2"
    >
      <span
        className="MuiTypography-root MuiTypography-body1 MuiListItemText-primary emotion-3"
      >
        myFileName
      </span>
    </div>
  </li>
</a>
`;
